tests/models/test_bedrock.py

```diff
@@ -95,6 +95,45 @@ async def test_bedrock_model(allow_model_requests: None, bedrock_provider: Bedro
     )
 
 
+async def test_bedrock_model_usage_limit_exceeded(
+    allow_model_requests: None,
+    bedrock_provider: BedrockProvider,
+):
+    model = BedrockConverseModel('us.anthropic.claude-sonnet-4-20250514-v1:0', provider=bedrock_provider)
+    agent = Agent(model=model)
+
+    with pytest.raises(
+        UsageLimitExceeded,
+        match='The next request would exceed the input_tokens_limit of 18 \\(input_tokens=19\\)',
+    ):
+        await agent.run(
+            'The quick brown fox jumps over the lazydog.',
+            usage_limits=UsageLimits(input_tokens_limit=18, count_tokens_before_request=True),
+        )
+
+
+async def test_bedrock_model_usage_limit_not_exceeded(
+    allow_model_requests: None,
+    bedrock_provider: BedrockProvider,
+):
+    model = BedrockConverseModel('us.anthropic.claude-sonnet-4-20250514-v1:0', provider=bedrock_provider)
```

Comment:
Would it work to not use the `us.` prefix in one of these tests? That would get us the test coverage the `coverage` job is complaining about.

If not, I'm OK with some `# pragma: no cover/branch`s in that function

Reply:
Unfortunately I can't test it right away either because of another access issue:

```py
E           botocore.errorfactory.AccessDeniedException: An error occurred (AccessDeniedException) when calling the CountTokens operation: User: X is not authorized to perform: bedrock:CountTokens on resource: Y because no identity-based policy allows the bedrock:CountTokens action
```

I'm OK with a separate test for the prefix removal helper, then, and both of these can keep their geo-qualified names.

Reply:
I returned the unit test